components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/listeners/UserManagementAuditLogger.java

```diff
@@ -522,4 +525,26 @@ private void maskClaimsInAuditLog(Map<String, String> claims, JSONObject data) {
             data.put(ListenerUtils.CLAIMS_FIELD, new JSONObject(claims));
         }
     }
+
+    /**
+     * Determines the appropriate audit message action based on the initiating persona in the current flow.
+     *
+     * @return The audit message action. If the flow is not available or the persona is unrecognized,
+     *         returns {@code ListenerUtils.CHANGE_PASSWORD_ACTION} by default.
+     */
+    private String getAuditMessageAction() {
```

Comment:
Since this is specific to password update context, lets denote that in the method name. Otherwise it can be confusing.